scripts/casestudies/schema.json

```diff
@@ -0,0 +1,281 @@
+{
+	"id":"adeogroup",
+	"company":{
+		"name":"Adeo Group",
+		"description":"Adeo owns different brands in retail industry focused on home improvement and DIY markets, like Leroy Merlin.",
+		"customers":"500M",
+		"industry":"Retail",
+		"revenue":"25.6B EURO turnover, including 768M EURO online.",
+		"website":"https://www.adeo.com/",
+		"logo":"/img/casestudies/adeo/logo.svg",
+		"contact":[
+			{
+				"name":"Ludovic Dussart",
+				"link":"https://twitter.com/ldussart"
+			}
+		]
+	},
+	"challenges":"Cost Component Repository product, part of the ADEO tech products, is used to calculate and publish transfer prices between different internal locations on the globe. Different business units use different information systems. It is hard to learn how each business unit share information about their systems, their API and accuracy.
+
+Initial solution was a developer portal with list of all applications and reference to dedicated documentation. Some legacy systems had docs written in MS Excel.
+
+There was a need for a standart way of describing event-driven architecture.
+",
```

Comment:
```suggestion
	"challenges":"Cost Component Repository product, part of the ADEO tech products, is used to calculate and publish transfer prices between different internal locations globally. Different business units use different information systems. Learning how each business unit shares information about its systems, API, and accuracy is hard. The initial solution was a developer portal with a list of all applications and reference to dedicated documentation. Some legacy systems had docs written in MS Excel. There was a need for a standard way of describing event-driven architecture.",
```

Reply:
shouldn't the extra spacing here be removed? wanted to double check before I did it for the rest in this file :)

Reply:
I'll fix all these issues if separate commit, thanks